docs/content/release_notes/whats_new.md

```diff
@@ -12,6 +12,10 @@ IBM® block storage CSI driver 1.8.0 now supports HyperSwap implementation, when
 
 This version adds NVMe/FC support for supported IBM Spectrum Virtualize family storage systems using Red Hat® Enterprise Linux® (RHEL) operating systems. 
 
+## Increased StorageClass `SpaceEfficiency` parameter capabilities
+
+Version 1.8.0 increases the `SpaceEfficiency` deduplication parameter options for IBM Spectrum Virtualize family storage systems. For more information, see [Creating a StorageClass](../content/configuration/creating_volumestrorageclass.md).
```

Comment:
this link doesn't work for me. I think removing the `content/` should fix it in this case.
there must be an automatic way to validate our links...
we rely on them heavily to avoid content duplications

Reply:
@matancarmeli7 could you take a look at https://github.com/gaurav-nelson/github-action-markdown-link-check for us?

Reply:
I will open a ticket for it, and will do it after we will finish with the GitHub actions PRs first

Reply:
CSI-3648

Reply:
thanks!

Reply:
fixed in docs_release-1.8.0 (PR 415)

Reply:
still [doesn't work](https://github.com/IBM/ibm-block-csi-driver/blob/17d919e94d523754830ebae450b82734ea9bfcec/docs/content/release_notes/whats_new.md#increased-storageclass-spaceefficiency-parameter-capabilities) for me
![image](https://user-images.githubusercontent.com/57443811/145577759-80003349-6990-40b0-a01a-53045e8ea21f.png)

Reply:
this was marked in docs-release-1.8.0 update for testing.